src/lua/serializer.h

```diff
@@ -184,6 +186,13 @@ void
 luaL_serializer_copy_options(struct luaL_serializer *dst,
 			     const struct luaL_serializer *src);
 
+/**
+ * Free memory allocated for serializer options.
+ * @param cfg Serializer configuration.
+ */
+void
+luaL_serializer_free_options(struct luaL_serializer *cfg);
```

Comment:
Nit: It seems, `luaL_serializer_destroy` name fits good here (see also the bottom of the [chapter about naming](https://www.tarantool.io/en/doc/latest/contributing/c_style_guide/#chapter-4-naming) in our C Style Guide). (If we free the options, but continue to use the serializer object, something is definitely wrong.) I would also zap the structure using the `TRASH` macro from `trivia/util.h` to reveal memory management/ownership problems.

Reply:
I've tried to apply the naming guide, and decided to name this one `luaL_serializer_delete`. Because it doesn't really destroys the serialezer it frees memory, used for it and it is also now used in options copy function:
```c
void
luaL_serializer_copy_options(struct luaL_serializer *dst,
			     const struct luaL_serializer *src)
{
	luaL_serializer_delete(dst);
	memcpy(dst, src, offsetof(struct luaL_serializer, end_of_options));
...
```

It would be strange to destroy the object to which we are going to copy something.

Reply:
As for me, `_delete` for a variable on a stack looks strange. My expectation is that I can't access this storage anymore after `_delete`, just like after the `delete` keyword in C++.

Reply:
> And renamed luaL_serializer_delete to luaL_serializer_options_delete.

OK for me. Resolving.